packages/amplication-git-utils/src/git/git-client.service.ts

```diff
@@ -0,0 +1,91 @@
+import { Injectable } from "@nestjs/common";
+import {
+  CreateRepository,
+  GitClient,
+  GithubFile,
+  GitProviderArgs,
+  Pagination,
+  PullRequest,
+  RemoteGitOrganization,
+  RemoteGitRepos,
+  RemoteGitRepository,
+  Repository,
+  File,
+} from "../types";
+import { prepareFilesForPullRequest } from "../utils/prepare-files-for-pull-request";
+import { GitFactory } from "./git-factory";
+
+@Injectable()
+export class GitClientService implements GitClient {
+  constructor(private readonly gitFactory: GitFactory) {}
+
+  async getGitInstallationUrl(
```

Comment:
why do you use async if you don't use await

Reply:
Because I return imminently